packages/cashier/src/pages/on-ramp/on-ramp.tsx

```diff
@@ -100,6 +100,8 @@ const OnRamp = ({
         onMountOnramp();
         if (typeof setSideNotes === 'function' && !is_switching && !is_loading) {
             setSideNotes([<OnRampSideNote key={0} />]);
+        } else {
+            setSideNotes([]);
```

Comment:
@hamid-deriv Could you move `setSideNotes([])` to the `willUnmount` part, I believe this issue happens because the previous sideNode doesn't remove from the screen in the `Unmount` phase so it's better to do the clean up there.

`if (typeof setSideNotes === 'function' && !is_switching && !is_loading) {
            setSideNotes([<OnRampSideNote key={0} />]);
        }

        return () => {
            onUnmountOnramp();
            setSideNotes([]);
        };`

Reply:
Yes, That's a better solution.